app-admin/sysstat/sysstat-12.2.1-r1.ebuild

```diff
@@ -43,6 +43,12 @@ src_prepare() {
 		done
 	fi
 
+	# bug #531032
+	if $(has_version sys-process/dcron); then
+		einfo dcron found, mangling crontab file
```

Comment:
What if the admin later switches to another crond implementation? How do you ensure it gets re-mangled?

Reply:
> What if the admin later switches to another crond implementation? How do you ensure it gets re-mangled?

Well, how do other packages which drop crontabs into /etc/cron.d handle that case? Do they usually at all?

Reply:
They don't mangle those files in the first place as only dcron has this incompatible crontab extension.

Wouldn't you rather solve this by telling users with sys-process/dcron installed that they should review the file?

Reply:
Alright thanks for the insight. I'll go the ewarn route then.

Would you like another PR for this, or should I just push to this one?